Solutions/Azure Activity/Analytic Rules/SubscriptionMigration.yaml

```diff
@@ -0,0 +1,60 @@
+id: 48c026d8-7f36-4a95-9568-6f1420d66e37
+kind: Scheduled
+name: Subscription moved to another tenant
+description: |
+  Detects when a subscription is moved to another tenant. This may indicate a subscription hijacking event.
+severity: High
+requiredDataConnectors:
+  - connectorId: AzureActivity
+    dataTypes:
+      - AzureActivity
+queryPeriod: 10m
+queryFrequency: 5m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+relevantTechniques:
+  - T1496
+query: |
+  let eventCapture = "moved from tenant ([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}) to tenant ([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})";
+  AzureActivity
+  | where OperationNameValue == "Microsoft.Subscription/updateTenant/action"
+  | extend EventCapture = extract_all(eventCapture, tostring(Properties_d.message))
+  | extend SourceTenant = iff(isnotempty(EventCapture), EventCapture[0][0], "")
+  | extend DestinationTenant = iff(isnotempty(EventCapture), EventCapture[0][1], "")
+  | extend UserParts = split(Caller, "@")
+  | extend UserName = iff(array_length(UserParts)==2, UserParts[0], Caller)
+  | extend UserUPNSuffix = iff(array_length(UserParts)==2, UserParts[1], "")
+  | extend Summary=Properties_d.message
```

Comment:
Summary is used in Alert Description. Can we convert this to string.
Also, this operation can be done once as we are converting it to string once at line 23.